backend/globaleaks/db/sqlite.sql

```diff
@@ -429,4 +429,18 @@ CREATE TABLE archivedschema (
     schema BLOB NOT NULL,
     UNIQUE (hash, type, language),
     PRIMARY KEY (id)
-)
+);
+
+CREATE TABLE log (
+    id INTEGER NOT NULL,
+    code TEXT NOT NULL,
+    args VARCHAR,
+    log_date TEXT NOT NULL,
+    subject TEXT NOT NULL,
+    log_message TEXT NOT NULL,
+    log_level INTEGER,
+    mail INTEGER,
```

Comment:
NULL is NULL (unassigned) and since now if i'm not wrong i've found that only = None = NULL